src/views/slideshow.c

```diff
@@ -560,78 +560,93 @@ int button_pressed(dt_view_t *self, double x, double y, double pressure, int whi
-  {
-    _set_delay(d, 1);
-    dt_control_log(ngettext("slideshow delay set to %d second", "slideshow delay set to %d seconds", d->delay), d->delay);
-  }
-  else if(key == GDK_KEY_Down || key == GDK_KEY_KP_Subtract || key == GDK_KEY_minus)
-  {
-    _set_delay(d, -1);
-    dt_control_log(ngettext("slideshow delay set to %d second", "slideshow delay set to %d seconds", d->delay), d->delay);
-  }
-  else if(key == GDK_KEY_Left)
-  {
-    if (d->auto_advance) dt_control_log(_("slideshow paused"));
-    d->auto_advance = FALSE;
-    _step_state(d, S_REQUEST_STEP_BACK);
-  }
-  else if(key == GDK_KEY_Right)
-  {
-    if (d->auto_advance) dt_control_log(_("slideshow paused"));
-    d->auto_advance = FALSE;
+    d->auto_advance = TRUE;
     _step_state(d, S_REQUEST_STEP);
   }
   else
   {
-    // go back to lt mode
     d->auto_advance = FALSE;
-    dt_ctl_switch_mode_to("lighttable");
+    dt_control_log(_("slideshow paused"));
   }
 
-  return 0;
+  return TRUE;
+}
+
+static gboolean _slow_down_callback(GtkAccelGroup *accel_group, GObject *acceleratable, guint keyval,
+                                    GdkModifierType modifier, dt_slideshow_t *d)
+{
+  _set_delay(d, 1);
+  dt_control_log(ngettext("slideshow delay set to %d second", "slideshow delay set to %d seconds", d->delay), d->delay);
+
+  return TRUE;
+}
+
+static gboolean _speed_up_callback(GtkAccelGroup *accel_group, GObject *acceleratable, guint keyval,
+                                   GdkModifierType modifier, dt_slideshow_t *d)
+{
+  _set_delay(d, -1);
+  dt_control_log(ngettext("slideshow delay set to %d second", "slideshow delay set to %d seconds", d->delay), d->delay);
+
+  return TRUE;
+}
+
+static gboolean _step_back_callback(GtkAccelGroup *accel_group, GObject *acceleratable, guint keyval,
+                                    GdkModifierType modifier, dt_slideshow_t *d)
+{
+  if (d->auto_advance) dt_control_log(_("slideshow paused"));
+  d->auto_advance = FALSE;
+  _step_state(d, S_REQUEST_STEP_BACK);
+
+  return TRUE;
+}
+
+static gboolean _step_forward_callback(GtkAccelGroup *accel_group, GObject *acceleratable, guint keyval,
+                                       GdkModifierType modifier, dt_slideshow_t *d)
+{
+  if (d->auto_advance) dt_control_log(_("slideshow paused"));
+  d->auto_advance = FALSE;
+  _step_state(d, S_REQUEST_STEP);
+
+  return TRUE;
+}
+
+static gboolean _exit_callback(GtkAccelGroup *accel_group, GObject *acceleratable, guint keyval,
+                               GdkModifierType modifier, dt_slideshow_t *d)
+{
+  // go back to lt mode
+  d->auto_advance = FALSE;
+  dt_ctl_switch_mode_to("lighttable");
+
+  return TRUE;
 }
 
 void init_key_accels(dt_view_t *self)
 {
   dt_accel_register_view(self, NC_("accel", "start and stop"), GDK_KEY_space, 0);
   dt_accel_register_view(self, NC_("accel", "exit slideshow"), GDK_KEY_Escape, 0);
+
+  dt_accel_register_view(self, NC_("accel", "slow down"), GDK_KEY_Up, 0);
```

Comment:
we had + and up and - and down, we will need to adjust the documentation.

Reply:
These can now be easily added by users who want it, so I thought it would be OK to remove as default. But could add back as well.

Reply:
Thinking more about this, yes probably better adding those shortcuts. TIA